doc/sphinx/source/ffi.rst

```diff
@@ -0,0 +1,12 @@
+Julia and Python Interfaces
+========================================
+
+libCEED provides high-level interfaces using the Julia and Python programming
+languages.
+
+More information about the Julia interface can be found at the `LibCEED.jl
+documentation <http://ceed.exascaleproject.org/libCEED-julia-docs/dev/>`_.
+
+Usage of the Python interface is illustrated through a sequence of `Jupyter
+Notebook
+tutorials <https://github.com/CEED/libCEED/tree/main/examples/tutorials>`_.
```

Comment:
Thanks for adding this. I don't know if it's any useful to reference libCEED's SciPy paper here or not. It has some details, but not even much in depth.

Reply:
Good idea, I added a link to the SciPy paper.

Reply:
Do we want to make an auto-documentation deployment for the Python interface and link it here? (Probably best in separate PR). It doesn't need to be fancy, but it would provide a reference for the function signatures.

Reply:
I think that sounds like a great idea. Yes, probably better in a separate PR